app/store/contact.go

```diff
@@ -90,3 +99,18 @@ func TelToName(tel string) string {
 	}
 	return tel
 }
+
+func TelUUID(tel string) (string, error) {
```

Comment:
Please use a more meaningful name here. `GetContactUUIDFromTel` for example. Though I don't get what that first `if` does for groups exactly so that name should even be completed.